plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/rest/TrinoRestCatalog.java

```diff
@@ -366,13 +374,37 @@ public void setTablePrincipal(ConnectorSession session, SchemaTableName schemaTa
     @Override
     public void createView(ConnectorSession session, SchemaTableName schemaViewName, ConnectorViewDefinition definition, boolean replace)
     {
-        throw new TrinoException(NOT_SUPPORTED, "createView is not supported for Iceberg REST catalog");
+        ViewBuilder builder =
+                restSessionCatalog.buildView(convert(session), toIdentifier(schemaViewName));
+        Schema schema = IcebergUtil.schemaFromViewColumns(typeManager, definition.getColumns());
+        Namespace defaultNamespace = Namespace.of(definition.getSchema().orElse(schemaViewName.getSchemaName()));
```

Comment:
remove `.orElse(`

if the view was created as 

```sql
-- no USE statement, so no default schema
CREATE VIEW iceberg_rest.default.view AS ...
```

the `ConnectorViewDefinition` will have `catalog` and `schema` fields empty and this must be preserved when retrieving it.

Please add a test showing that ConnectorViewDefinition round-trips. This should go into BaseTrinoCatalogTest as a prep commit and should cover cases when ConnectorViewDefinition has `catalog` and `schema` fields set and also the case when `catalog` and `schema` fields are empty.